### ISSUE TITLE
Replace CTF flag-action texts

### ISSUE THREAD

**Issue by GingerAdonis** [MrGreenGaming/MTA-Resources#58]
CTF flag-action texts should say 'Your team' instead of '<team color> team' when it applies to your team.

For example:
"Red's flag has been taken by SDK" should be something like
"Your team has captured the Red flag."

Thoughts?

**Comment by AleksCore** [comment]
Sure, it should be like this in my opinion

**Comment by EsDeKa** [comment]
I agree, it always felt like there was something annoying about those messages

**Comment by JarnoVgr** [comment]
If a flag returns to the base it still shows the text in old style.
